torchtune/utils/metric_logging.py

```diff
@@ -166,9 +176,29 @@ def __init__(
             group=group,
             reinit=True,
             resume="allow",
-            config=kwargs,
+            **kwargs,
         )
 
+    def log_config(self, config: OmegaConf) -> None:
+        "Logs the config to W&B. Also updates config on overview tab."
+
+        resolved = OmegaConf.to_container(config, resolve=True)
+        self._wandb.config.update(resolved)
+        
+        output_config_fname = Path(os.path.join(
+            config.checkpointer.checkpoint_dir, 
+            f"torchtune_config_{self._wandb.run.id}.yaml"
+            )
+        )
+        OmegaConf.save(config, output_config_fname)
+        try:
+            
+            print(f"Logging {output_config_fname} to W&B under Files")
```

Comment:
Use the debug logger instead of print:
```
# At the top right after imports and before class definitions
log = utils.get_logger("DEBUG")
...
# In methods
log.info(...)
```